notebooks/samples/101 - Adult Census Income Training.ipynb

```diff
@@ -67,7 +67,8 @@
    "source": [
     "from mmlspark import TrainClassifier\n",
     "from pyspark.ml.classification import LogisticRegression\n",
-    "model = TrainClassifier(model=LogisticRegression(), labelCol=\" income\", numFeatures=256).fit(train)"
+    "model = TrainClassifier(model=LogisticRegression(), labelCol=\" income\", numFeatures=256).fit(train)\n",
```

Comment:
hmm not sure why these file changes are here, strange, is this rebased to latest master?  These changes are already in master right?